src/domain/Invoice.ts

```diff
@@ -0,0 +1,54 @@
+type Double = number;
+
+export class Invoice {
+  id: string;
+  type: string;
+  amount: Double;
```

Comment:
@daniszcode aqui não seria **number**??

Reply:
Eu criei um type Double que é number, eu posso continuar usando assim ou não é uma boa prática? é melhor colocar logo como number?

Reply:
Acho que nesse caso podemos deixar o number direto. A menos que esse Double for ter alguma caracteristica especifica não coberta pelo number.